Unite/Unite/Controllers/ChatController.cs

```diff
@@ -29,4 +29,11 @@ public ChatController(AssistantService assistantService)
         
         return Ok(response);
     }
+
+    [HttpGet("health")]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    public IActionResult HealthCheck()
+    {
+        return Ok();

```

Comment:
It is the worst ever. No more buddy!

Reply:
Thank you for your feedback. Could you please elaborate on what specifically is not functioning as expected? This will enable us to address the issue more effectively.